pandas/core/frame.py

```diff
@@ -137,16 +137,16 @@
 """
 
 _merge_doc = """
-Merge DataFrame objects by performing a database-style join operation by
-columns or indexes.
+Merge DataFrame or named Series objects by performing a database-style join
+operation by columns or indexes.
 
 If joining columns on columns, the DataFrame indexes *will be
 ignored*. Otherwise if joining indexes on indexes or indexes on a column or
 columns, the index will be passed on.
 
 Parameters
 ----------%s
-right : DataFrame, Series or dict
```

Comment:
As of my prior commit initially this only said  `right : DataFrame`, not sure when the part around `dict` was added. Have removed mention of `dict` as this PR only ensures that `DataFrame` and `Series` can be merged

Reply:
Have not tested merge with dict - as recently changed local machine - is this needed in this PR?